packages/coingecko/src/helpers.ts

```diff
@@ -0,0 +1,21 @@
+import 'whatwg-fetch'
```

Comment:
Why adding whatwg-fetch?

Reply:
was feel not possible to test on node which don't have native `fetch` support

works now with mocha + jsdom so the whatwg-fetch import could be removed
https://stackoverflow.com/questions/34059644/mocha-command-giving-referenceerror-window-is-not-defined